### PR TITLE
Fix #32

### LINKED ISSUE #32: 0.11 has wrong __version__
Note: this is the same as hgrecco/pint#1038.

Pint v0.11 has the version string "0.0.0".

**Steps to reproduce:**
```
conda create -n pint11 -c conda-forge "pint==0.11"
conda activate pint11
python -c "import pint; print(pint.__version__)"
```
Output: `0.0.0`

**Additional context:**
Other versions have the correct version string.
```
conda create -n pint10 -c conda-forge "pint==0.10"
conda activate pint10
python -c "import pint; print(pint.__version__)"
```
Output: `0.10`

```
conda create -n pinthead pip python
conda activate pinthead
pip install git+https://github.com/hgrecco/pint
python -c "import pint; print(pint.__version__)"
```
Output: `0.12.dev8+g3aba60e`

### PR DESCRIPTION
**Pull request by dgasmith**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dgasmith** [comment]
Alright, I give up. This works fine locally, but not here.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,15 +13,17 @@ source:
 
 build:
     noarch: python
-    script: {{ PYTHON }} -m pip install . --no-deps -vv
-    number: 0
+    script:
+      - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+      - {{ python }} -m pip install . --no-deps
```
Comment (scopatz) [inline recipe/meta.yaml]: Try putting this on a single line: 

```
SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ python }} -m pip install . --no-deps
```

Comment (dgasmith) [reply recipe/meta.yaml]: Tried that one [here](https://github.com/conda-forge/pint-feedstock/pull/34/commits/2381eaa7cc26b9ea5bdf89ff38b83424389e4679).

Comment (scopatz) [reply recipe/meta.yaml]: :man_shrugging:

Comment (dgasmith) [reply recipe/meta.yaml]: Yea, I feel the same way. I think the short is we are going to recommend everyone to stay away from `setuptools_scm` for now.

Comment (scopatz) [reply recipe/meta.yaml]: Yeah, I agree. It is pretty annoying to deal with as a downstream packager

Comment (dgasmith) [reply recipe/meta.yaml]: Needed `toml`... either way success!

**Comment by dgasmith** [comment]
@dopplershift Finally got this to work, please squash merge when you have the chance.

**Pull request merged**